src/components/Property/Types/NumericProperty/util.ts

```diff
@@ -0,0 +1,9 @@
+export function stepToDecimalPlaces(step: number): number {
+  const stepStr = step.toString();
+  const decimalPlaces = stepStr.includes('.') ? stepStr.split('.')[1].length : 0;
```

Comment:
Are you getting the last element of the array with the 1 index? how about `pop()` ?
```suggestion
  const decimalPlaces = stepStr.includes('.') ? stepStr.split('.')[1].length : 0;
```